context/app/static/js/hooks/useAbortableEffect.js

```diff
@@ -0,0 +1,24 @@
+// https://www.debuggr.io/react-update-unmounted-component/
+// https://github.com/reactjs/rfcs/issues/137
+import { useEffect } from 'react';
+
+function useAbortableEffect(effect, dependencies) {
```

Comment:
We should use this for any effect around an async? I think either ilan or mark wrote a eslint rule for something special we'd like to enforce, and this might be another case: it didn't seem that complicated. (But that's certainly not a priority.)

Reply:
Filed #1054